src/server/NetworkingServer.cpp

```diff
@@ -129,16 +152,8 @@ void NetworkingServer::ApiServer() {
                         sf::Uint32 api_command;
                         switch (client.receive(packet)) {
                             case sf::Socket::Done:
-                                if (packet >> api_command && (api_command == (sf::Uint32)APICommand::register_client)) {
-                                    sf::Packet response_packet;
-                                    sf::Uint32 api_command = (sf::Uint32)APICommand::register_client;
-                                    if(response_packet << api_command << mClientIdCounter << (sf::Uint32) mCurrTick) {
-                                        client.send(response_packet);
-                                        std::cout << "Received client registration. Issued client ID " << mClientIdCounter << std::endl;
-                                        mClientPositions[mClientIdCounter] = new float[2]{0.f, 0.f};
-                                        mClientSocketsToIds[&client] = mClientIdCounter;
-                                        mClientIdCounter++;
-                                    }
+                                if (packet >> api_command && api_command == (sf::Uint32)APICommand::register_client) {
```

Comment:
nice, thanks for pulling RegisterClient and Move into their own methods, much cleaner